addons/web/static/src/views/list/list_renderer.js

```diff
@@ -81,7 +81,7 @@ export class ListRenderer extends Component {
             columns: this.getActiveColumns(this.props.list),
         });
         this.withHandleColumn = this.state.columns.some((col) => col.widget === "handle");
-        useExternalListener(document, "click", this.onGlobalClick.bind(this));
+        useExternalListener(document, "click", this.onGlobalClick.bind(this), { capture: true });
```

Comment:
I'm also fixing a similar issue in another branch and I also considered this. I think we did that in the past, I can't remember why we don't anymore. I'll dig a little and come back to you. But this is indeed typically what we would like here.

Reply:
Tests are indeed failing for a good reason. I didn't check all of them, but there's already a flow that can't work with this: add a line in an editable list view (main view, not x2many), click on Save => it will discard the row instead of saving it

Reply:
It's definitely not an easy change, especially in stable.

Reply:
It is a bit tricky to fix, maybe revert the original commit that stops the event propagation and try to solve in another way ? wdyt ?

Reply:
Or we decide not to fix this in stable. It's not a critical nor blocking issue. It's annoying, but that's it. I'll let you decide. Removing the `.stop` modifier from autocomple is fine, especially considering why it has been added there in the first place. So if you can find a workaround for accounting, fine by me.

I'll create a master task to try to define the globalClick handler in capture, which is the right thing to do imo.

Reply:
```diff
--- a/addons/web/static/src/views/list/list_renderer.js
+++ b/addons/web/static/src/views/list/list_renderer.js
@@ -81,7 +81,13 @@ export class ListRenderer extends Component {
             columns: this.getActiveColumns(this.props.list),
         });
         this.withHandleColumn = this.state.columns.some((col) => col.widget === "handle");
-        useExternalListener(document, "click", this.onGlobalClick.bind(this), { capture: true });
+        useExternalListener(document, "click", this.onGlobalClick.bind(this));
+        /**
+         * using mouse down just because when the view has autocomplete along with a list
+         * when selecting an element from the autocomplete we prevent the event propagation of the `click`
+         * this is only for stable (up to 18.0) and will be handled properly in master
+         */
+        useExternalListener(document, "mousedown", this.onGlobalClick.bind(this));
         this.tableRef = useRef("table");
```

Wdyt about this ? using `mousedown` event just for the case of autocomplete clicked, will do this up to 18.0 and let it be fixed properly in master.

Reply:
This'll break tests as well I think. It probably breaks the "Save" button clicked flows.

Reply:
@aab-odoo , as far as i saw, we dont need anymore that commit because now we have "Search more", please let know if i'm wrong.

Reply:
maybe @h4818 can confirm?

Reply:
@aab-odoo @walidsahli I can't recall the exact reason why that was important for the analytic_distribution widget.
I tested the widget - and it seems to work well (both with the `.stop` and without ie this branch)
maybe something else changed which fixes the case I was trying to cover :man_shrugging: 
either way... seems ok to me to remove it. If there is an issue we will resolve it with a test.